test/functional/rpc_blockchain.py

```diff
@@ -388,6 +390,33 @@ def _test_gettxoutsetinfo(self):
         # Unknown hash_type raises an error
         assert_raises_rpc_error(-8, "'foo hash' is not a valid hash_type", node.gettxoutsetinfo, "foo hash")
 
+    def _test_gettxout(self):
+        self.log.info("Test gettxout")
```

Comment:
This logs seems a bit vague, a more descriptive log would be useful

Reply:
Yeah, I agree. I was following the same log structure used by other tests in this file. I changed it to `"Validating gettxout RPC response"`. Let me know if that's better. Thanks